packages/libs/react-ui/src/components/Box/Box.tsx

```diff
@@ -0,0 +1,44 @@
+import { Sprinkles, sprinkles } from '../../styles';
+
+import React, { createElement, ElementType } from 'react';
+
+export interface IBoxProps extends Sprinkles {
```

Comment:
Can we try making the type more strict? So there is a typescript error if anyone tries to pass in anything other than the margin props